packages/tanstack-react-router-urql/jsr.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@urql/tanstack-react-router",
+  "version": "1.0.0",
+  "exports": {
+    ".": "./src/index.ts"
+  },
+  "exclude": [
+    "node_modules",
+    "cypress",
```

Comment:
Does not exist in this folder, nor do the others

Reply:
done